dev/eda-cr/eda-k8s-ing.yml

```diff
@@ -27,6 +27,8 @@ spec:
   extra_settings:
     - setting: EDA_MAX_RUNNING_ACTIVATIONS
       value: '11'
+    - setting: EDA_ALLOW_LOCAL_RESOURCE_MANAGEMENT
```

Comment:
There are two more files in `dev/` they should not have this setting too? in the end all are standalone deployments.